### DISCUSSION TITLE
Will all css files be packed in the entry file?  Why is this？

### DISCUSSION THREAD

**Discussion by Deja-vuuu** [vercel/next.js#41314] [Help]
Will all css files be packed in the entry file?  Why is this？

I found my home page loaded css also loaded the css of other pages css did not do code cutting?

**Comment by icyJoseph** [comment]
Are you using CSS modules?

> In production, all CSS Module files will be automatically concatenated into many minified and code-split .css files. These .css files represent hot execution paths in your application, ensuring the minimal amount of CSS is loaded for your application to paint.

Note that CSS modules concatenates into many minified and code-split css files.

Stylesheets loaded in `_app` are also concatenated into a large CSS file. 

If you are using Tailwind, this is to be expected as well, all of the minified, compressed and purged CSS is compiled into one block.
